pkg/fanal/handler/misconf/misconf_test.go

```diff
@@ -80,44 +87,158 @@ func Test_FindingFSTarget(t *testing.T) {
 			wantErr: true,
 		},
 		{
-			input:      []string{"/"},
-			wantTarget: "/",
+			input:      []string{string(os.PathSeparator)},
+			wantTarget: string(os.PathSeparator),
 			wantPaths:  []string{"."},
 		},
 		{
-			input:      []string{"/home/user"},
-			wantTarget: "/home/user",
+			input:      []string{filepath.Join(string(os.PathSeparator), "home", "user")},
+			wantTarget: filepath.Join(string(os.PathSeparator), "home", "user"),
 			wantPaths:  []string{"."},
 		},
 		{
-			input:      []string{"/home/user", "/home/user/something"},
-			wantTarget: "/home/user",
+			input: []string{
+				filepath.Join(string(os.PathSeparator), "home", "user"),
+				filepath.Join(string(os.PathSeparator), "home", "user", "something"),
+			},
+			wantTarget: filepath.Join(string(os.PathSeparator), "home", "user"),
 			wantPaths:  []string{".", "something"},
 		},
 		{
-			input:      []string{"/home/user", "/home/user/something/else"},
-			wantTarget: "/home/user",
+			input: []string{
+				filepath.Join(string(os.PathSeparator), "home", "user"),
+				filepath.Join(string(os.PathSeparator), "home", "user", "something", "else"),
+			},
+			wantTarget: filepath.Join(string(os.PathSeparator), "home", "user"),
 			wantPaths:  []string{".", "something/else"},
 		},
 		{
-			input:      []string{"/home/user", "/home/user2/something/else"},
-			wantTarget: "/home",
+			input: []string{
+				filepath.Join(string(os.PathSeparator), "home", "user"),
+				filepath.Join(string(os.PathSeparator), "home", "user2", "something", "else"),
+			},
+			wantTarget: filepath.Join(string(os.PathSeparator), "home"),
 			wantPaths:  []string{"user", "user2/something/else"},
 		},
 		{
-			input:      []string{"/foo", "/bar"},
-			wantTarget: "/",
+			input: []string{
+				filepath.Join(string(os.PathSeparator), "foo"), filepath.Join(string(os.PathSeparator), "bar"),
+			},
+			wantTarget: string(os.PathSeparator),
 			wantPaths:  []string{"foo", "bar"},
 		},
 		{
-			input:      []string{"/", "/bar"},
-			wantTarget: "/",
+			input:      []string{string(os.PathSeparator), filepath.Join(string(os.PathSeparator), "bar")},
+			wantTarget: string(os.PathSeparator),
 			wantPaths:  []string{".", "bar"},
 		},
 	}
 
 	for _, test := range tests {
 		t.Run(fmt.Sprintf("%#v", test.input), func(t *testing.T) {
+
+			if runtime.GOOS == "windows" {
```

Comment:
Seems like it already skips Windows. Do we need these lines?
https://github.com/aquasecurity/trivy/blob/0c056195650e38aa166c360412ebfe1b013f4510/pkg/fanal/handler/misconf/misconf_test.go#L75-L77